kotlin/src/main/kotlin/tyler/shapes/Shape.kt

```diff
@@ -0,0 +1,11 @@
+package tyler.shapes
+
+abstract class Shape {
+    abstract fun computeArea(): Double
```

Comment:
Because `computeArea` is a function that doesn't actually implement the function, that is what makes it `abstract`. We have no information to compute the area, so all we can do is say that obviously a shape should have a function to compute the area: all shapes have an area.